debian/changelog

```diff
@@ -1,3 +1,9 @@
+securedrop-client (0.15.0~rc1) unstable; urgency=medium
```

Comment:
Instead of having a duplicate entry with the same version, you can update the timestamp in the previous entry.